tests/browser/heda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -0,0 +1,92 @@
+*** Settings ***
+
+Resource        tests/HEDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Go To Heda Home
+    Go To Heda Home
```

Comment:
The keyword being defined is being called in its definition causing all tests to error.  The first line of the definition should not call the keyword being defined.

Also, what is the purpose of this test case?  It seems like something that is more a keyword than a test case.  If there's a need to do this in every test it should be a HEDA library keyword that is called at the start of the test cases that need it.

Reply:
Correct - this is needed to get into the HEDA tab area.